src/Components/Artwork/SaveButton/useSaveArtworkToLists.ts

```diff
@@ -15,6 +15,11 @@ type Artwork = {
   isInAuction: boolean
   isSavedToDefaultList: boolean
   isSavedToCustomLists: boolean
+  collectorSignals: {
+    auction: {
+      lotWatcherCount: number
+    } | null
+  } | null
```

Comment:
#non-blocking: this `type Artwork` looks like it should be pulling from an auto-generated relay file, vs being hand rolled